### PR TITLE
chore: (hack11) rename renderWithWrappersTL

### PR DESCRIPTION
**Pull request by patrinoua**
This PR renames 
renderWithWrappers => renderWithWrappersLEGACY
renderWithWrappersTL => renderWithWrappers

### PR Checklist

- [ ] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

### CODE REVIEW

File: src/app/screens/HomeTabs/Albums/AlbumArtworks.tests.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { GlobalStore } from "app/store/GlobalStore"
 import { act } from "react-test-renderer"
 import { flushPromiseQueue } from "shared/tests/flushPromiseQueue"
```
Comment (rajsam003) [inline src/app/screens/HomeTabs/Albums/AlbumArtworks.tests.tsx:3]: Can we also remove this?

**Comment by rajsam003** [comment]
We dont have this in energy
`renderWithWrappers => renderWithWrappersLEGACY`
So maybe we can remove that from the PR description. 
Other than that it's a good change.

**Review by pvinis — APPROVED** [review]

File: src/app/screens/HomeTabs/Albums/AlbumArtworks.tests.tsx (hunk 1 above)
Comment (patrinoua) [reply src/app/screens/HomeTabs/Albums/AlbumArtworks.tests.tsx:3]: 👀 hmm the point of this pr is just renaming renderWithWrappersTL not general refactoring

Comment (rajsam003) [reply src/app/screens/HomeTabs/Albums/AlbumArtworks.tests.tsx:3]: So do we need another PR to just remove a line that is not in use? 😂
I see that we missed removing it before. I dont see anything wrong with removing it now.

Comment (patrinoua) [reply src/app/screens/HomeTabs/Albums/AlbumArtworks.tests.tsx]: huhu removed!

**Review by rajsam003 — APPROVED** [review]
Awesome. Thanks 🙏🏽

**Pull request merged**